data/ch/team.yml

```diff
@@ -0,0 +1,46 @@
+############################### team ############################
+team:
+  enable : true
+  title : Our Team
+  team_member :
+    # team member loop
+    - name : Benjamin Bengfort
+      image_webp : images/team/benjamin.webp
+      image : images/team/benjamin.jpg
+      designation : Founder/CEO
+      content : Vision
+      social :
+        - icon : "ti-linkedin" # themify icon pack : https://themify.me/themify-icons
+          link : "https://www.linkedin.com/in/bbengfort/"
+        - icon : "ti-twitter-alt" # themify icon pack : https://themify.me/themify-icons
+          link : "https://twitter.com/bbengfort"
+        - icon : "ti-github" # themify icon pack : https://themify.me/themify-icons
+          link : "https://github.com/bbengfort"
+
+    # team member loop
+    - name : Rebecca Bilbro
+      image_webp : images/team/rebecca.webp
+      image : images/team/rebecca.jpg
+      designation : Founder/CTO
+      content : Produit
+      social :
+        - icon : "ti-linkedin" # themify icon pack : https://themify.me/themify-icons
+          link : "https://www.linkedin.com/in/rebeccabilbro/"
+        - icon : "ti-twitter-alt" # themify icon pack : https://themify.me/themify-icons
+          link : "https://twitter.com/rebeccabilbro"
+        - icon : "ti-github" # themify icon pack : https://themify.me/themify-icons
+          link : "https://github.com/rebeccabilbro"
+
+    # team member loop
+    - name : Edwin Schmierer
+      image_webp : images/team/edwin.webp
+      image : images/team/edwin.jpg
+      designation : Advisor
```

Comment:
We can remove this file (we'll add Chinese-language in a future PR using the go-18n for Hugo approach)